joomla/joomla-cms
# [com_fields] Inherit url scheme restriction from fields plugin

Pull Request for comment https://github.com/joomla/joomla-cms/pull/13319#issuecomment-271272419.

### Summary of Changes
Checks if the parameter is NULL and then fetches the parameter from the plugin. Additionally the default scheme value is removed to allow any scheme initially.

### Testing Instructions
- In the Fields -> Url Plugin, the scheme should be set to HTTPS.
- Create an url field and leave the scheme empty.

### Expected result
When editing an article, only urls with the scheme https should be allowed.
